eng/CodeGeneration.targets

```diff
@@ -50,3 +56,4 @@
 
 
 
+
```

Comment:
Nit: is the auto-updater adding a blank line to the end everytime it updates? Might be good to stop that eventually.